.github/workflows/dev-mode-layer-publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish Dev Mode Extension
+on:
+  push:
```

Comment:
I think it should rather be published after all tests pass, and ideally, if we also establish some release flow (as in the case of node packages)